lib/helpers/setupNetlifyFunctionForPage.js

```diff
@@ -47,8 +48,11 @@ const setupNetlifyFunctionForPage = ({
     });
   });
 
+  // Copy any dynamic import chunks
+  copyDynamicImportChunks(functionDirectory);
+
   // Copy page
-  const nextPageCopyPath = join(functionDirectory, "nextPage.js");
+  const nextPageCopyPath = join(functionDirectory, "nextPage", "index.js");
```

Comment:
The reason behind this change is to organize all the page files in the same directory, or is it also required for the fix to work?

Reply:
yeah, what jason said :( the lambda generated by next (that we previous renamed to nextPage.js to pair with next_whatever_the_page_name_is.js) is searching for the chunks one level up at `../` 💀